flink-batch-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/split/ParameterValuesProvider.java

```diff
@@ -15,17 +15,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.flink.api.java.io.jdbc;
+package org.apache.flink.api.java.io.jdbc.split;
 
-import java.io.OutputStream;
+import java.io.Serializable;
+
+import org.apache.flink.api.java.io.jdbc.JDBCInputFormat;
 
-@SuppressWarnings("unused")
 /**
- * Utility class to disable derby logging
- */
-public class DerbyUtil {
-	public static final OutputStream DEV_NULL = new OutputStream() {
-		public void write(int b) {
-		}
-	};
+ * 
+ * This interface is used by the {@link JDBCInputFormat} to compute the list of parallel query to run (i.e. splits).
+ * Each query will be parameterized using a row of the matrix provided by each {@link ParameterValuesProvider} implementation
+ * 
+ * */
+public interface ParameterValuesProvider extends Serializable {
```

Comment:
The `ParameterValuesProvider` does not need to be `Serializable`. However, all values in the parameter values array must be, otherwise the JdbcInputFormat cannot be serialized. So the signature of `getParameterValues()` should be adapted to `public Serializable[][] getParameterValues()`.